e2e/playwright/search/src/tests/search-results-general.spec.ts

```diff
@@ -61,21 +53,20 @@ test.describe('Search Results - General', () => {
     }
   });
 
+  test.beforeEach(async ({ loginPage }) => {
+    await Utils.tryLoginUser(loginPage, username, username, 'beforeEach failed');
+  });
+
   test.afterAll(async () => {
-    try {
-      await trashcanApi.emptyTrashcan();
-      await nodesApi.deleteCurrentUserNodes();
-    } catch (error) {
-      console.error(`afterAll failed: ${error}`);
-    }
+    await Utils.deleteNodesSitesEmptyTrashcan(nodesApi, trashcanApi, 'afterAll failed', undefined, undefined, undefined);
```

Comment:
instead of passing 3 undefined you can pass nothing as these parameters are optional